docs/source/sdk.rst

```diff
@@ -42,25 +42,37 @@ Creating SDK
 Creating wallet contract
 ------------------------
 
-create
-^^^^^^
+createFutureWallet
+^^^^^^^^^^^^^^^^^^
 
-**sdk.create(ensName)**
+**sdk.createFutureWallet()**
 
-  creates new wallet contract.
+prepares wallet contract deployment.
 
-  Parameters:
-    - **ensName** : string - choosen ENS name with existing domain (ENS domain supported by relayer)
-  Returns:
-    `promise`, that resolves to a pair ``[privateKey, contractAddress]``, where:
+Returns:
+  `promise`, that resolves to ``{privateKey, contractAddress, waitForBalance, deploy}``, where:
 
-    - *privateKey* - private key assigend to the wallet contract for signing future transactions
-    - *contract address* - address of newly deployed wallet contract, with choosen ENS name assigned
+  - *privateKey* - private key assigned to the wallet contract for signing future transactions
+  - *contract address* - address of newly deployed wallet contract, with chosen ENS name assigned
+  - *waitForBalance* - function that waits for contract address balance change in acceptable by relayer token
```

Comment:
function that waits for contract address balance change in a way that will allow the wallet contract to be deployed.